src/mpc-hc/MainFrm.cpp

```diff
@@ -11189,6 +11189,10 @@ void CMainFrame::OpenSetupAudio()
 {
     m_pBA->put_Volume(m_wndToolBar.Volume);
 
+    if (AfxGetAppSettings().nCLSwitches & CLSW_VOLUME) {

```

Comment:
Possible race condition due to that line getting hit from the graph thread before the toolbar volume has initialised?